<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>passwordDialog</class>
 <widget class="QDialog" name="passwordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PassMan - Add new credentials</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="accountField">
     <property name="toolTip">
      <string>Enter account username/E-mail address</string>
     </property>
     <property name="placeholderText">
      <string>Enter account username/E-mail address</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QTextEdit" name="noteField">
     <property name="toolTip">
      <string>Enter relevant note (optional)</string>
     </property>
     <property name="placeholderText">
      <string>Enter relevant note (optional)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="dialogButtons">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Account:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Note:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Title:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="titleField">
     <property name="toolTip">
      <string>Enter title for this entry</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Enter title for this entry</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="generateRandom">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Generate random password</string>
        </property>
        <property name="text">
         <string>Generate random password</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="copyPassword">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Copy Password To Clipboard</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLineEdit" name="passwordField">
        <property name="toolTip">
         <string>Enter password</string>
        </property>
        <property name="maxLength">
         <number>96</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
        <property name="placeholderText">
         <string>Enter password</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="togglePassword">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Toggle Password Visibility</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="2">
       <widget class="QSpinBox" name="passwordLength">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Set length of password to generate</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>96</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>noteField</zorder>
  <zorder>groupBox</zorder>
  <zorder>dialogButtons</zorder>
  <zorder>label_4</zorder>
  <zorder>titleField</zorder>
  <zorder>accountField</zorder>
 </widget>
 <tabstops>
  <tabstop>titleField</tabstop>
  <tabstop>accountField</tabstop>
  <tabstop>passwordField</tabstop>
  <tabstop>togglePassword</tabstop>
  <tabstop>generateRandom</tabstop>
  <tabstop>passwordLength</tabstop>
  <tabstop>noteField</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>dialogButtons</sender>
   <signal>accepted()</signal>
   <receiver>passwordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogButtons</sender>
   <signal>rejected()</signal>
   <receiver>passwordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
